Samples/AudioPreprocessing/AudioPreprocessing/Model/PreprocessModel.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace AudioPreprocessing.Model
+{
+    public class PreprocessModel
+    { 
+        private List<string> repositoryData;
+        private string melSpecPath;
+        public string ImportantInfo
```

Comment:
is there a more descriptive name than "important info"? what is the info?

Reply:
This is a residue from a tutorial I followed. Cleaned it up.